craco.config.js

```diff
@@ -0,0 +1,8 @@
+// craco.config.js
+module.exports = {
+  style: {
+    postcss: {
+      plugins: [require("tailwindcss"), require("autoprefixer")],
+    },
+  },
```

Comment:
Job well done!

You added this file for a reason, if I may ask, why?

It is weird to me to see this file because app initialized with create-react-app won't generate this.

Reply:
Hello Bellom, I added that because I want to use Tailwind for my UI. I followed Tailwind instructions here: https://tailwindcss.com/docs/guides/create-react-app 
The application was made with create-react-app.